### PR TITLE
Add btsieve support for ledger networks

### LINKED ISSUE #815: expose network on btsieve
As part of btsieve's query, client should be able to specify the desired network (mainnet, testnet, etc).

btsieve should reject the HTTP create query if the network is not supported (ie, btsieve connected to testnet only but query for mainnet)

### PR DESCRIPTION
**Pull request by bonomat**
btsieve supports now a ledger network e.g. 
`/queries/bitcoin/regtest/*` or `/queries/ethereum/regtest/*`, ... 

I decided for a path param as the `network` is part of the ledger and not part of the query, i.e. if you create a query for `/queries/ethereum/transactions` with an additional field or a query param, we would need to check on the btsieve side manually while using the path param a 404 is automatically returned if the network is not supported. 

This network is part of the ledger configuration in the config file of btsieve. 

Resolves #815

### CODE REVIEW

File: api_tests/harness.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ class LedgerRunner {
                 `Waiting ${wait_time} for ${to_be_started.join(", ")} to start`
             );
 
-            await util.sleep(wait_time);
+            await util.sleep(10000);
```
Comment (LLFourn) [inline api_tests/harness.js]: :^)

PS If the wait_time is not long enough (locally or remotely) increase the number in the toml file.

Comment (bonomat) [reply api_tests/harness.js]: wupsi.. that should not be in this commit 🤦‍♂️

File: application/btsieve/src/bin/btsieve.rs [hunk 2]
```diff
@@ -61,6 +66,11 @@ fn create_bitcoin_routes(
         settings.node_password.as_str(),
     );
 
+    // TODO fix this unwrap
+    let blockchain_info = bitcoin_rpc_client.get_blockchain_info().unwrap().unwrap();
```
[the remark this answers is not in the document]
Comment (bonomat) [reply application/btsieve/src/bin/btsieve.rs]: I _fixed_ it by failing with an error for now. Let's fix this with #557 .

File: application/btsieve/src/bin/btsieve.rs [hunk 3]
```diff
@@ -132,6 +146,13 @@ fn create_ethereum_routes(
         Http::new(settings.node_url.as_str()).expect("unable to connect to Ethereum node");
     let web3_client = Arc::new(Web3::new(transport));
 
+    // TODO remove this unwrap here
+    let network = web3_client.net().version().wait().unwrap();
```
[the remark this answers is not in the document]
Comment (bonomat) [reply application/btsieve/src/bin/btsieve.rs]: Same as above.

**Pull request merged**